website/docs/tutorials/08-cairo-1-support/03-cheatcodes/call.md

```diff
@@ -24,5 +32,34 @@ fn test_call_simple() {
     assert(*return_data2.at(0_u32) == 25, 'check call result');
 }
 ```
+## Handling call failures
+```cairo title="Deployed contract"
+#[contract]
+mod MinimalContract {
+    #[view]
+    fn panic_with(panic_data: Array::<felt252>) {
+        panic(panic_data);
+    }
+}
+```
+
+```cairo title="Test"
+use cheatcodes::RevertedTransactionTrait;
+
+#[test]
+fn test_call_errors() {
+    let mut panic_data = ArrayTrait::new();
+    panic_data.append(2); // Array length
+    panic_data.append('error');
+    panic_data.append('data');
+    
+    match call(deployed_contract_address, 'panic_with', panic_data) {
+        Result::Ok(x) => assert(false, 'Shouldnt have succeeded'),
+        Result::Err(x) => {
+            assert(x.first() == 'error', 'first datum doesnt match');
+            assert(*x.panic_data.at(1_u32) == 'data', 'second datum doesnt match');
```

Comment:
Is this correct? isn't `panic_data` like this: `[2, 'error', 'data']`?

Reply:
No, the format is as in old cairo, array len and then array items